src/layout-pack.js

```diff
@@ -82,6 +105,12 @@ var PackLayout = React.createClass({
     },
 
     componentDidUpdate: function() {
+        // When image finish loading run reposition
```

Comment:
Could we add a boolean property to enable this? I'm fine setting it to `true` as default, but we need a way to disable this behavior for users that don't want it.